package.json

```diff
@@ -24,11 +24,15 @@
     "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4"
+    "pg-format": "^1.0.4",
+    "supertest": "^6.2.3"
   },
   "dependencies": {
     "dotenv": "^16.0.1",
-    "pg": "^8.7.3"
+    "express": "^4.18.1",
+    "node": "^17.7.2",
+    "pg": "^8.7.3",
+    "postgres": "^3.2.2"
```

Comment:
why have you installed Postgres? what does this do??

Reply:
Not sure, i was just having problems getting the db to be rquired in the test file, turns out it was the order that i was requiring files that was the issue, but i installed pg first thinking this was the problem?